src/edge-agent/connectionsManager/ConnectionsManager.ts

```diff
@@ -254,7 +253,8 @@ export class ConnectionsManager implements ConnectionsManagerClass {
   async sendMessage(message: Message): Promise<Message | undefined> {
     message.direction = MessageDirection.SENT;
     await this.pluto.storeMessage(message);
-    return this.mercury.sendMessageParseMessage(message);
+    await this.mercury.sendMessageParseMessage(message);
+    return message;
```

Comment:
Earlier in time we had a workaround to avoid the user's getting an exception when the response of the didcomm endpoint is null.

Parsed message is null and now it won't crash because null really means "delivered". This though just avoids the exception on the user side.

Reply:
Solves this, just as n example: https://github.com/input-output-hk/atala-prism-wallet-sdk-ts/issues/101